continuous-integration.md

```diff
@@ -0,0 +1,29 @@
+Continuous Integration
+====
+
+Continuous Integration (CI) is a practice where developers frequently push their code into a shared repository. 
+Automated builds and checks run on each check-in, allowing teams to detect problems early.
+
+You should have continuous integration set up in your repository running the following tasks where relevant:
+* Running unit and integration tests
+* Building your application
+* Linting your codebase
+* Publishing artifacts
+* Other appropriate tasks for your repository
+
+## Platforms
+
+* Use TeamCity to run continuous integration tasks
+* If possible, have a centralised script in the repository named `scripts/ci`  
+    - This adheres to GitHub's [Scripts To Rule Them All pattern](https://github.com/github/scripts-to-rule-them-all)
+* For public repositories, you can use [GitHub Actions](https://docs.github.com/en/actions) for some above tasks, but we are yet to develop a safe way of uploading artifacts to RiffRaff through it
```

Comment:
Should we be suggesting this if there isn't a safe way to do it? Probably we need to discuss what is mean't by safe.

Reply:
Sorry, didn't see this comment until now! This is a good point, but I think this could be better explained by changing the phrasing a bit. 

I've just pushed a change that aims to rephrase the recommendation from "there isn't a safe way to push artefacts to RiffRaff" to "we haven't developed a best practice for this yet", which is more accurate. I think, at the time, I meant "safe" as "you can't safely use the recommended `node-riffraff-artifact` in GitHub Actions as  you'd have to publish/configure credentials somewhere", but this would be fairly safe to do if you configured AWS credentials in each repo and rotated them regularly.

I think a good  (and safe)solution for this is possible (likely relying on Guardian org-wide tokens being set up and a custom GitHub Action to implement it), and I think it's sensible to still encourage people to use GitHub Actions as it is a great interface, free for public repositories, and safe to use for the tasks mentioned besides publishing artefacts to RiffRaff.